web/tests/functional/source_change/test_source_change.py

```diff
@@ -114,3 +114,38 @@ def test_store(self):
 
         ret = codechecker.store(self._codechecker_cfg, 'test_proj')
         self.assertEqual(ret, 1)
+
+    def test_store_config_file_mod(self):
+        """Storage should be successful if a non report related file changed.
+
+        Checking the modification time stamps should be done only for
+        the source files mentioned in the report plist files.
+        Modifying any non report related file should not prevent the storage
+        of the reports.
+        """
+        test_proj = os.path.join(self.test_workspace, 'test_proj')
+
+        ret = codechecker.log(self._codechecker_cfg, test_proj,
+                              clean_project=True)
+        self.assertEqual(ret, 0)
+
+        ret = codechecker.analyze(self._codechecker_cfg, test_proj)
+        self.assertEqual(ret, 0)
+
+        test_proj_path = self._testproject_data['project_path']
+
+        # touch one file so it will be analyzed again
+        touch(os.path.join(test_proj_path, 'null_dereference.cpp'))
```

Comment:
Please try to store the results to the server after you change this file and check that the storage will fail.

Reply:
We have a separate test for this use case:
https://github.com/Ericsson/codechecker/blob/45b9ce31d47c57119de87aef2fdcf0295cbb6fac/web/tests/functional/source_change/test_source_change.py#L89-L93

It tests if a file mentioned in a plist report changed since the plist report was generated. In that case the store should fail.

Reply:
@gyorb Great, I didn't see it.